lms/djangoapps/courseware/views/views.py

```diff
@@ -268,14 +269,13 @@ def courses(request):
     # Add marketable programs to the context.
     programs_list = get_programs_with_type(request.site, include_hidden=False)
 
-    # Add upcoming webinars in context to display them on `learn` page.
     return render_to_response(
         "courseware/courses.html",
         {
             'courses': courses_list,
             'course_discovery_meanings': course_discovery_meanings,
             'programs_list': programs_list,
-            'webinars': Webinar.objects.filter(status=Webinar.UPCOMING).order_by('start_time'),
+            'webinars': Webinar.objects.filter(is_cancelled=False, start_time__gte=now()).order_by('start_time'),
```

Comment:
How about showing the current webinars too? Like as a user, let us say I got late for a minute or two, I won't able to see the webinar I was going to join. Though user can get link from the email, but we should facilitate this case on the platform too. So
`start_end__gte=now`
@asadali145

Reply:
I guess they will receive reminders and all that stuff so it shouldn't be much of a problem. If you still think we should add some time limit. What time limit do you suggest for this? @ahmed-faraz46

Reply:
I still feel we should add, because our platform is a primary source to access the webinars, and the time is `end_date`, so we just need to change the filter param. @asadali145

Reply:
@asadali145 Ahmed has suggested to use end time instead of start time. Meaning if an event starts at 11:00 AM and ends at 1:00 PM, it should remain in the upcoming tab till 1:00 PM instead of 11:00 AM. And I think it's really well pointed out by Ahmed 👏

Reply:
So what I have understood is like we should show those webinars in which `end_time__gte=now()`. Right?

Reply:
Got it. Thanks @harris-irfan @ahmed-faraz46. Lets implement this @muak50 as suggested by Ahmed.

Reply:
Yes.

Reply:
@ahmed-faraz46 @harris-irfan @asadali145 Sure, Will do in updated commit.